src/actions/index.js

```diff
@@ -3,7 +3,7 @@ export const CREATE_BOOK = (book) => ({
   payload: book,
 });
 
-export const REMOVE_BOOK = (id) => ({
+export const REMOVE_BOOK = (book) => ({
```

Comment:
- [ ] [OPTIONAL] Rename the actions so they follow the camelCase naming convention.
Only the types to dispatch(EX: 'const REMOVE_BOOK = 'REMOVE_BOOK';') should be written in all uppercase because they are constants.
Name the actions as normal functions instead.